designdocs/passkey_p256.md

```diff
@@ -0,0 +1,556 @@
+        const passkey_pk = registerPassKey.publicKey; // obtained from the registration response
+
+        // Sign Passkey Public Key using Account Key, this is used to prove ownership of the account
+        const account_sig_passkey_pk = account_keypair.sign(passkey_pk);
+      ```
+
+5. **Backup options offered to the user**:
+   - The user is prompted to download the seed phrase or backup the seed phrase to a secure location.
+
+6. **Browser JavaScript transfers information to Web Wallet Backend**:
+   - The following data is sent:
+     - `passkey_pk`
+     - `account_pk`
+     - `account_sig_passkey_pk`
+     - `passkey_sig_account_pk`
+
+7. **Web Wallet validates and stores the data**:
+   - The backend validates the signatures to ensure ownership and integrity.
+   - The backend stores the following:
+     - `passkey_pk`
+     - `account_pk`
+     - `account_sig_passkey_pk`
+     - Optionally, `passkey_sig_account_pk` can be stored for account recovery.
+
+### Transaction Submission Specification
+
+Browser/Client receives the following data from the backend:
+
+- `passkey_pk`
+- `account_pk`
+- `account_sig_passkey_pk`
+
+1. **Transaction Formation**:
+   - The client initiates a transaction by providing the details needed for the transaction (e.g., recipient address, amount, nonce).
+   - The transaction payload includes the actual tx and proof of account ownership.
+
+    ```javascript
+        // Form the transaction payload
+
+        // an example transaction
+        const tx = api.tx.balances.transfer(..., ...);
+
+        // Prepare the passkey payload to be signed by Passkey
+        const account_ownership_proof = {
+          Sr25519: account_sig_passkey_pk,
+        };
+
+        let passkeyCallData = {
+          accountId: account_pk,
+          accountOwnershipProof: account_ownership_proof,
+          call: tx,
+        };
+
+        let passkeyCall = api.registry.createType('PalletFrequencyTxPaymentPasskeyCall', passkeyCallData);
+    ```
+
+2. **Passkey Signing**:
+   - The transaction payload is sent to the Passkey for signing.
+   - The Passkey signs the transaction payload using its private key, producing a `passkey_sig_tx`.
+
+    ```javascript
+
+      // Sign the passkeyCall payload using Passkey
+      const passkeyTxSignature = await SimpleWebAuthnBrowser.startAssertion({
+        challenge: passkeyCall.toU8a(),
+        // Additional options
+      });
+
+      // The signature generated by Passkey
+      passkey_sig_tx = passkeyTxSignature.signature;
+      passkey_authenticator_data = passkeyTxSignature.authenticatorData;
+      passkey_client_data = passkeyTxSignature.clientData;
+      //
+    ```
+
+3. **Transaction Submission**:
+   - The transaction payload, that requires, `passkey_pk`, `passkey_sig_tx`, `passkey_authenticator_data`, `passkey_client_data` is submitted to the chain.
+   - The transaction is submitted to the chain as in `unsigned` extrinsic.
```

Comment:
```suggestion
   - The transaction payload (consisting of `passkey_pk`, `passkey_sig_tx`, `passkey_authenticator_data`, `passkey_client_data`) is submitted to the chain.
   - The transaction is submitted to the chain as an `unsigned` extrinsic.
```

Reply:
Also, might want a "see note" here for the "submitted as an unsigned extrinsic", and link to the discussion point later on about tradeoffs between unsigned vs. signed extrinsics

Reply:
Good suggestion, Added a link to discussion  :+1: 
> The transaction is submitted to the chain as an `unsigned` extrinsic.